### PR TITLE
Minutes of the last board meeting

### PR DESCRIPTION
**Pull request by martha-thomae**
This pull request contains the minutes of the January 2022 board meeting. Please let me know if everything is in order :)

### CODE REVIEW

File: community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Present: Johannes Kepper, Anna E. Kijas, KlausRettinghaus, Benjamin W. Bohl, Stefan Münnich, Laurent Pugin, Perry Roland, Martha E. Thomae
+
+Regards: Kristina Richts
+
+Minutes: Martha E. Thomae and Stefan Münnich
+
+## Administrative Chair & Technical Co-Chairs
+- Anna Kijas was appointed as Administrative chair.
+- The technical co-chairs are Benjamin Bohl and Stefan Münnich.
```
Comment (musicEnfanthen) [inline community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md:16]:
```suggestion
- The technical co-chairs are Benjamin W. Bohl and Stefan Münnich.
```

File: community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: "2022-01-19 Virtual MEI Board Meeting"
+---
+
+# 2022-01-19 Virtual MEI Board Meeting
+
+Present: Johannes Kepper, Anna E. Kijas, KlausRettinghaus, Benjamin W. Bohl, Stefan Münnich, Laurent Pugin, Perry Roland, Martha E. Thomae
```
Comment (musicEnfanthen) [inline community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md:8]:
```suggestion
Present: Benjamin W. Bohl, Johannes Kepper, Anna E. Kijas, Klaus Rettinghaus, Stefan Münnich, Laurent Pugin, Perry Roland, Martha E. Thomae
```

File: community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Regards: Kristina Richts
+
+Minutes: Martha E. Thomae and Stefan Münnich
+
+## Administrative Chair & Technical Co-Chairs
+- Anna Kijas was appointed as Administrative chair.
+- The technical co-chairs are Benjamin Bohl and Stefan Münnich.
+- Johannes Kepper was appointed as treasurer.
```
Comment (musicEnfanthen) [inline community/mei-board/protocols/2022-01-19_Virtual-Board-Meeting.md:17]:
```suggestion

## Current Financial Situation

Financial situation is stable. We will have a continuous income into the MEI pocket for at least the following two years.

## Future Conferences

There are expressions of interest for MEC 2023 and 2024. Details to be announced at the given time.

## ODD Meetings

The ODD meetings will still be held at the usual schedule:
- last Friday of odd months
- last Thursday of even months

```

**Comment by musicEnfanthen** [comment]
Thank you @martha-thomae . Some additions above.

**Comment by martha-thomae** [comment]
Thanks for noticing these, @musicEnfanthen!

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thank you @martha-thomae !

@annakijas1 Perhaps you would like to do the final check?

**Review by bwbohl — APPROVED** [review]

**Pull request merged**